azure-pipelines.yml

```diff
@@ -1,28 +1,77 @@
-# Android
-# Build your Android project with Gradle.
-# Add steps that test, sign, and distribute the APK, save build artifacts, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/languages/android
-
+variables:
+  sdk: 'iphoneos'
+  configuration: 'Build'
+  msMapsModulesPath: 'iOS/MSMapsModules'
+  msMapsModulesToRootPath: '../../'
+  msMapsModulesWorkspacePath: '$(msMapsModulesPath)/MSMapsModules.xcworkspace'
+  testAppKeysPath: '$(msMapsModulesPath)/MSMapsModulesTestApp/keys.plist'
+  referenceAppPath: 'iOS/MSMapsModulesReferenceApp'
+  referenceAppWorkspacePath: '$(referenceAppPath)/MSMapsModulesReferenceApp.xcworkspace'
+  
 trigger:
 - master
 
-pool:
-  vmImage: 'windows-latest'
-
-steps:
-
-  # Create dummy secrets.gradle
-- bash: |
-    echo ext.credentialsKey = "\"DummyKey\"" > secrets.gradle
-    echo `<secrets.gradle`
-  displayName: Create secrets.gradle
-  workingDirectory: 'Android/Samples/app'
+jobs:
+- job: Build_Android
+  pool:
+    vmImage: 'windows-latest'
+  steps:
+    # Create dummy secrets.gradle
+  - bash: |
+      echo ext.credentialsKey = "\"DummyKey\"" > secrets.gradle
+      echo `<secrets.gradle`
+    displayName: Create secrets.gradle
+    workingDirectory: 'Android/Samples/app'
+  - task: Gradle@2
+    inputs:
+      workingDirectory: 'Android/Samples'
+      gradleWrapperFile: 'Android/Samples/gradlew'
+      gradleOptions: '-Xmx3072m'
+      publishJUnitResults: false
+      testResultsFiles: '**/TEST-*.xml'
+      tasks: 'build'
 
-- task: Gradle@2
-  inputs:
-    workingDirectory: 'Android/Samples'
-    gradleWrapperFile: 'Android/Samples/gradlew'
-    gradleOptions: '-Xmx3072m'
-    publishJUnitResults: false
-    testResultsFiles: '**/TEST-*.xml'
-    tasks: 'build'
+- job: Build_iOS
+  pool:
+    vmImage: 'macOS-10.14'
+  steps:
+  - bash: |
+      cd $(msMapsModulesPath)
```

Comment:
Prefer using the workingDirectory parameter to bash instead of changing the directory in the script. Set it per bash command that requires a specific working directory.

Reply:
The Podfile contains `project './MSMapsModules.xcodeproj'` and when I was testing locally, I changed directory to this folder and then ran pod install. To use the workingDirectory parameter instead of changing directory, I'll also change that line in the Podfile to use the same path. Is it alright to change the path in the Podfile also?